UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs

```diff
@@ -65,6 +67,84 @@ void InitAdminController()
 		LoadBanList();
 		LoadCurrentAdmins();
 		LoadWhiteList();
+		AdminLogStart();
+	}
+
+	private void AdminLogStart()
+	{
+		adminLogPath = Path.Combine(Application.streamingAssetsPath, "admin", "adminlog1.txt");
+		adminLog2Path = Path.Combine(Application.streamingAssetsPath, "admin", "adminlog2.txt");
+
+		if (!File.Exists(adminLogPath))
+		{
+			//Creates file
+			var file = File.CreateText(adminLogPath);
+
+			//Todo add to file round number here
+			file.WriteLine("true");
+
+			file.Close();
+		}
+
+		if (!File.Exists(adminLog2Path))
+		{
+			//Creates file
+			var file = File.CreateText(adminLog2Path);
+
+			//Todo add to file round number here
+			file.WriteLine("false");
+
+			file.Close();
+		}
+
+		var first = File.ReadLines(adminLogPath).First() == "true";
+
+		var second = File.ReadLines(adminLog2Path).First() == "true";
```

Comment:
why is it writing "true" / "false" to the files?

Reply:
Because for some reason even though this script is on a manager the bool way of changing which file is written to, as i wanted to keep a prev round log file resets on map change so theres no way to tell it to change file, so i added the true / false to the files instead.

Reply:
Humm might be able to fix it now player list wasnt on the manager i thought it was on.

Reply:
Think i'll make a logging manager, also be good for player logging when thats added.

Reply:
Cant seem to get the variable to persistent through round restarts so im gonna have to stickm with this method for now.

Reply:
@ThatDan123 I don't think a variable or writing these values should be necessary to implement rolling logs. Just maintain 2 log files for example - currentRound.log, prevRound.log. At the start of each round, if currentRound.log exists, copy it and overwrite prevRound.log if it exists.